src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,7 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService member = new UserService();
+        return member.findByEmail(email) != null && member.findByPassword(password) != null;
```

Comment:
is the logic here correct?) Can there be such a situation that the copier did not enter his password, but that password exists in the database with another user. In this case, your program will skip the user and it will not be very true. I suggest limiting yourself to one getByEmail method and getting the password from the User and comparing it with what was entered at the entrance)